dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -291,6 +291,11 @@ internal open class RumViewScope(
             attributes.putAll(event.attributes)
             stopped = true
             sendViewUpdate(event, writer)
+            // there will no longer be updates on this View so we can
```

Comment:
@0xnm  not sure when to do this cleanup, if I do it here I am afraid that some ongoing resources might still need this information later...